Gruntfile.js

```diff
@@ -13,10 +13,9 @@ module.exports = function(grunt) {
           ['stringify', {
             extensions: ['.bpmn']
           }],
-          ['babelify', {
-            presets: 'env',
-            global: true
-          }]
+        ],
+        plugin: [
+          'esmify'
```

Comment:
Is it _really_ necessary to move away from `babelify`? `esmify` seems to be nice, but it's a tiny project and I feel like keeping the huge community of `babelify` would be advantageous.

Reply:
Unfortunately yes. It does not work with babel because babel is set up to transpile the nose modules, too. This breaks because if some dependency of the properties panel. Esmify is just required to transpile the imports. Setting up babel correctly is a longer term issue imo and i will take care of it eventually.

Reply:
Okay, fair enough. But then we can surely get rid of the `babelify` dependency. 🙂

Reply:
Yes. That should work.